src/SystemMessage/index.js

```diff
@@ -27,15 +27,15 @@ const SystemMessage = (props: Props) => (
     <Message type={props.type}>{props.children}</Message>
```

Comment:
looks broken: 
![image](https://user-images.githubusercontent.com/7850592/39251524-d01eaca2-48a3-11e8-8a2c-b8e66ac82371.png)

Reply:
Will fix it.